src/test/java/com/google/api/codegen/testdata/java_mock_service_impl_library.baseline

```diff
@@ -43,37 +43,38 @@ import com.google.protobuf.Empty;
 import com.google.protobuf.GeneratedMessage;
 import io.grpc.stub.StreamObserver;
 import java.util.ArrayList;
+import java.util.LinkedList;
 import java.util.List;
+import java.util.Queue;
 
 @javax.annotation.Generated("by GAPIC")
 public class MockLibraryServiceImpl implements LibraryService  {
   private ArrayList<GeneratedMessage> requests;
-  private ArrayList<GeneratedMessage> responses;
+  private Queue<GeneratedMessage> responses;
 
   public MockLibraryServiceImpl() {
     requests = new ArrayList<>();
-    responses = new ArrayList<>();
+    responses = new LinkedList<>();
   }
 
   public List<GeneratedMessage> getRequests() {
     return requests;
   }
 
   public void setResponses(List<GeneratedMessage> responses) {
```

Comment:
The code below could be further streamlined by the making this method take varargs, and then using `Lists.newLinkedList(Arrays.asList(responses))`: https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#asList(T...)

This would make the code below look cleaner, since a single response is the most common case.